packages/ui/src/i18n/dictionaries/en.ts

```diff
@@ -16,6 +16,8 @@ export const enDict = {
   Email: 'Email',
   'Enter your code': 'Enter your code',
   'Enter your username': 'Enter your username',
+  'Enter your phone number': 'Enter your phone number',
+  'Enter your email': 'Enter your email',
```

Comment:
We don't have translations for this for other languages yet...

Reply:
I think that's fine, we need to rely on contributors for translations. We can start asking Discord community soon!

Reply:
👍